style.css

```diff
@@ -69,3 +123,295 @@ h1 span {
+  width: 695vh;
+  height: 220px;
+  margin-left: 10%;
+  margin-top: 50px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+}
+
+.container2 h3 {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  font-size: 24px;
+  color: #3a4a42;
+  margin-left: 80px;
+  padding-top: 10px;
+  margin-bottom: 10px;
+  line-height: 32px;
+}
+
+.container2 ul {
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.container2 a button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 15px;
+  margin-left: 115px;
+  margin-bottom: 48px;
+  margin-top: 48px;
+  color: #fff;
+  border: #fff;
+  gap: 10px;
+  width: 122px;
+  height: 48px;
+  bottom: 8px;
+  background: #36b37f;
+  border-radius: 8px;
+}
+
+.container2 a button:hover {
+  color: #1e488f;
+}
+
+.container2 .second-img {
+  background: url(images/snapa.png) no-repeat;
+  width: 295px;
+  height: 220px;
+  margin-left: 10%;
+  margin-top: 0;
+}
+
+.container2 .third-img {
+  background: url(images/Snapb.png) no-repeat;
+  width: 295px;
+  height: 220px;
+  margin-left: 10%;
+  margin-top: 0;
+}
+
+.container2 .fourth-img {
+  background: url(images/Snapc.png) no-repeat;
+  width: 295px;
+  height: 220px;
+  margin-left: 10%;
+  margin-top: 50px;
+}
+
+.container2 .fifth-img {
+  background: url(images/snapa.png) no-repeat;
+  width: 295px;
+  height: 220px;
+  margin-left: 10%;
+  margin-top: 0;
+}
+
+.container2 .sixth-img {
+  background: url(images/Snapb.png) no-repeat;
+  width: 295px;
+  height: 220px;
+  margin-left: 10%;
+  margin-top: 0;
+}
+
+.container3 h2 {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  font-size: 40px;
+  position: absolute;
+  font-weight: 700;
+  margin-left: 80px;
+  color: #172b4d;
+  margin-top: -65px;
+}
+
+.container3 p {
+  font-size: 20px;
+  display: flex;
+  text-align: center;
+  line-height: 1.5;
+  color: #42526e;
+  padding-top: 15px;
+}
+
+.my-resume {
+  display: flex;
+  align-items: center;
+  color: #fff;
+  border: #fff;
+  gap: 10px;
+  width: 150px;
+  height: 48px;
+  bottom: 8px;
+  padding: 10px 10px;
+  margin-left: 110px;
+  background: #36b37f;
+  letter-spacing: 1px;
+  border-radius: 4px;
+}
+
+.last-img img {
+  width: auto;
+  height: auto;
+  margin-left: 290px;
+}
+
+.container3 a {
+  padding: 20px;
+}
+
+.last-img2 img {
+  display: flex;
+  position: absolute;
+  margin-top: -80px;
+  margin-left: -13px;
+}
+
+.container3 .my-resume:hover {
+  color: #1e488f;
+}
+
+.container4 {
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  background: #ebf0ee;
+  padding-bottom: 84px;
+  padding-left: 36px;
+  padding-right: 36px;
+  margin-bottom: 24px;
+}
+
+.container4 .last-sec {
+  background: url(images/icon-languages.png) no-repeat;
+  width: 295px;
+  min-height: 130px;
+  margin-left: 30%;
+  margin-top: 40px;
+}
```

Comment:
- [ ] According to the project requirement, you must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).I would kindly suggest you to align all technology under the skills to fit the figma design and this will make your website look professional. 

- [ ] Your design
![Screenshot 2023-06-10 112445](https://github.com/AhmedMh0/Mobile-Portfolio/assets/92458236/dc908056-f209-4a91-a46f-90067754d807)

- [ ] Figma design
![Screenshot 2023-06-10 112344](https://github.com/AhmedMh0/Mobile-Portfolio/assets/92458236/ad2d9710-c2a0-4dd5-b835-194a9c54f1f6)